front/src/common/redux/auth/auth.reducer.js

```diff
@@ -4,7 +4,7 @@ import { getUserType } from '../../functions';
 const initialState = {
     isLoggedIn: false,
     errorMessage: '',
-    role: getUserType(),
+    //role: getUserType(),
```

Comment:
Please, remove commented line if you don't need it

Reply:
Outdated